### PR TITLE
Review Generate OVA section

### PR DESCRIPTION
**Pull request by javimed**
## Description
This PR does the following on the [Development/Wazuh packages generation guide/Virtual machine](https://documentation.wazuh.com/current/development/packaging/generate-ova.html) section.
- Adds a recommendation for hardware requirements to generate the OVA file.
- Adds editorial changes

### Docs building
- [X] Compiles without warnings.
### Code formatting and web optimization
- [X] Uses three spaces indentation.
- [X] Adds or updates meta descriptions accordingly.
- [X] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [X] Uses present tense, active voice, and semi-formal registry.
- [X] Uses short, simple sentences.
- [X] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by davidcr01 — APPROVED** [review]

File: source/development/packaging/generate-ova.rst [hunk 1]
```diff
@@ -3,60 +3,66 @@
...
 
-This will generate a Virtual machine with the Wazuh manager |WAZUH_CURRENT_OVA| and ELK |ELASTICSEARCH_LATEST| installed using production packages.
+      $ ./generate_ova.sh -h
 
-.. code-block:: console
+   .. code-block:: none
+      :class: output
 
-  # ./generate_ova.sh -r dev
+      General usage: generate_ova.sh [OPTIONS]
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:58]: The [generate_ova.sh](https://github.com/wazuh/wazuh-packages/blob/v4.5.1/ova/generate_ova.sh) script also includes a `staging` option, we should describe it in the documentation. 
```
DESCRIPTION
        -r,  --repository
                Use development or production repository.
                Values: [prod|dev|staging]. By default: prod.
```

File: source/development/packaging/generate-ova.rst [hunk 2]
```diff
@@ -3,60 +3,66 @@
...
+-  `Virtual Box <https://www.virtualbox.org/manual/UserManual.html#installation>`__
+-  `Vagrant <https://www.vagrantup.com/docs/installation/>`__
+-  `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+-  `Python <https://www.python.org/download/releases/2.7/>`__
 
- * `Virtual Box <https://www.virtualbox.org/manual/UserManual.html#installation>`_
- * `Vagrant <https://www.vagrantup.com/docs/installation/>`_
- * `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
- * `Python <https://www.python.org/download/releases/2.7/>`_
+We recommend using a system with at least the following specifications.
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:19]:
```suggestion
We recommend using a system with at least the following hardware specifications:
```

File: source/development/packaging/generate-ova.rst [hunk 3]
```diff
@@ -3,60 +3,66 @@
...
 
-Download our wazuh-packages repository from GitHub and go to the ova directory.
+-  8 GB RAM
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:21]: For consistency with other sections of the documentation, consider using a table to specify the hardware requirements.

**Review by s-ocando — COMMENTED** [review]
Hi @javimed, please review my suggestions below.

File: source/development/packaging/generate-ova.rst (hunk 1 above)
Comment (javimed) [reply source/development/packaging/generate-ova.rst]: The staging option doesn't seem to be useful for the general users.

**Review by s-ocando — APPROVED** [review]

**Pull request merged**